assets/chat/css/style.scss

```diff
@@ -1333,66 +1333,84 @@ hr {
 }
 
 #chat-whisper-users {
-  .content > ul {
-    margin: 0;
-    padding: $gutter-md 0;
-  }
-  .conversation {
-    list-style: none;
-    position: relative;
-    cursor: pointer;
-    padding-left: $gutter-md;
-    display: flex;
-    align-items: center;
-    .user {
-      color: $color-accent;
-      display: block;
-    }
-    .user:hover {
-      color: $color-accent-light;
-      .badge {
-        color: $color-chat-text1;
+  .section {
```

Comment:
I find this CSS challenging to read and I think it's a combination of there being no newlines before each nested ruleset and excessive nesting. For instance, you don't need this level of specificity to select `.unread`.
```
#chat-whisper-users .section .user-entry .right .unread {
}
```
I think the issue is you're trying to make the nesting in your CSS reflect the nesting in your markup. Besides readability, you can also run into situations where you have to rely on `!important` more often to override styles.